### ISSUE TITLE
XACML-TEST

### ISSUE THREAD

**Issue by jainh** [att/XACML#24]
I tried to run maven package, two test cases in Xacml-Test are failing one is ResponseTest and FunctionDefinitionAccessPermittedTest. I fixed ResponseTest but looking at another, do you want me to create pull request ?

**Comment by pdragosh** [comment]
Yes, please go ahead and and do the pull request. I need to investigate the FunctionDefinitionAccessPermittedTest, it isn't implemented but certainly should fail.

**Comment by jainh** [comment]
ok, i will do that, Just to let you know, I am interested in contributing to this project and we do see lots of improvement that can be added to the code. Do you have any design document ? are you now actively committing to this project or not?

**Comment by pdragosh** [comment]
The best I can do is point you to the wiki as far as design document. Sorry wish I had more. We are actively committing to this project now. I am currently trying to mavenize it going forward, working in the "develop" branch. Trying to merge with master hopefully this week or early next week. And do an official first release. After that we are open to suggestions for improvement.

**Comment by jainh** [comment]
Good to hear "you are actively committing", I do have issue in running PAP-admin ui using jetty in Intellij, I had to add jetty plugin and only 9.0 or below version. I would suggest make it simple (Like you already have parent-child pom model) but also add jetty plugin for web based application to facilitate running and avoid dependencies from IDE (like Eclipse), You have META-INF dispersed in Webcontent which might not be the right place to load JPA configuration.
